test/malli/swagger_test.cljc

```diff
@@ -362,6 +362,15 @@
                                      400 {:schema (m/schema ::error-resp
                                                             {:registry registry})}}})))))
 
+  (testing "no schema in responses ignored"
+    (is (= {:definitions nil
+            :responses {200 {:description "" :schema {:type "string"}}}}
+           (swagger/swagger-spec {#_::swagger/parameters
+                                  #_{:body {:description "just a body"}}
+                                  ::swagger/responses
+                                  {500 {:description "fail"}
```

Comment:
I think the result should have a `500 {:description "fail"}`. Cf. https://github.com/metosin/reitit/blob/master/test/cljc/reitit/swagger_test.clj#L173